src/net_processing.cpp

```diff
@@ -4622,6 +4629,7 @@ bool PeerManagerImpl::SendMessages(CNode* pto)
                         // Fetch the top element from the heap
                         std::pop_heap(vInvTx.begin(), vInvTx.end(), compareInvMempoolOrder);
                         uint256 hash = vInvTx.back();
+                        bool flood_tx = pto->m_tx_relay->m_txs_to_announce.at(hash);
```

Comment:
3692db74f86186789c9dad4005f7b768721d5afe

Move to inner scope, make const.

Reply:
Making const. Can't move because it's erased in two lines after, so it won't be accessible in the inner scope.
And I can't call `.second` on `.erase`, because it returns a number of erased elements.

One alternative could be to use `txid_to_ann_protocol` in `vInvTx` (instead of uint256). But then I won't be able to reuse `CompareInvMempoolOrder` in `AnnounceTxs`.